cogs/csgo.py

```diff
@@ -297,9 +297,9 @@ async def pug(self, ctx: commands.Context, *args):
 
         # TODO: Add check for EU/CIS flag
         if len(team1_flags) > 0:
-            team1_country = max(k for k, v in Counter(team1_flags).items() if v > 1)
+            team1_country = Counter(team1_flags).most_common(1)[0]
```

Comment:
```suggestion
team1_country = Counter(team1_flags).most_common(1)[0][0]
```
or
```suggestion
team1_country, _ = Counter(team1_flags).most_common(1)[0]
```
I believe what you have now will return a tuple of (flag_name, count). You should index into it again, or use tuple unpacking.

Reply:
Fixed it there. I'm actually a dumb programmer sometimes lmao